build.yaml

```diff
@@ -417,24 +431,26 @@ common_cli:
     # which are made available through a webserver running on port 80.
     # We pass the URL of our Jenkins Slave to the acceptance test nodes
     # through the --build-server parameter below.
-    parse_logs \$venv/bin/python admin/run-acceptance-tests \
+    parse_logs \${venv}/bin/python admin/run-acceptance-tests \
     --distribution \${DISTRIBUTION_NAME} \
     --provider rackspace --dataset-backend rackspace \
     --branch \${TRIGGERED_BRANCH} \
     --build-server  \
     http://\$(wget -qO- http://instance-data/latest/meta-data/public-ipv4)  \
     --config-file /tmp/acceptance.yaml \
-    \${ACCEPTANCE_TEST_MODULE} ; RC=\$?
+    \${ACCEPTANCE_TEST_MODULE}
+    JOB_EXIT_STATUS="\$( updateExitStatus \$? )"
 
```

Comment:
make sure JOB_EXIT_STATUS will contain 0 if every thing is working fine until now